### PR TITLE
chore: create .gitpod.yml

### PR DESCRIPTION
**Pull request by EdamAme-x**
Close https://github.com/honojs/hono/issues/2866

### CODE REVIEW

**Comment by EdamAme-x** [comment]
by Astro
https://github.com/withastro/astro/blob/main/.gitpod.yml

**Comment by EdamAme-x** [comment]
`.gitpod/gitpod-setup.sh` near `./.devcontainer/Dockerfile`

**Comment by EdamAme-x** [comment]
@yusukebe what do you think? Should I erase line breaks for indentation like astro?

File: .gitpod.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+tasks:
+  - name:  Setup
+    init: yarn instal --pure-lockfile
```
Comment (yusukebe) [inline .gitpod.yml:3]: It's no problem to use `yarn`, but we recommend using the `bun` command.

**Review by yusukebe — APPROVED** [review]
LGTM!

**Comment by yusukebe** [comment]
@EdamAme-x 

Looks good! Thank you!

**Pull request merged**